lib/nexmo_developer/app/services/translator/smartling/api/download_file.rb

```diff
@@ -18,7 +18,13 @@ def build_request
         def uri
           @uri ||= begin
             uri = URI("https://api.smartling.com/files-api/v2/projects/#{@project_id}/locales/#{@locale_id}/file")
-            uri.query = URI.encode_www_form({ 'fileUri' => @file_uri, 'retrievalType' => 'published' })
+            uri.query = URI.encode_www_form(
+              {
+                'fileUri' => @file_uri,
+                'retrievalType' => 'published',
+                'includeOriginalStrings' => false,
```

Comment:
@bencgreenberg do we still need the `includeOriginalStrings` flag?

Reply:
No, we don't, great catch! Let me get rid of it, one sec.